app/javascript/packs/application.js

```diff
@@ -1,5 +1,6 @@
 
 import React from 'react'
+import ReactDOM from 'react-dom'
 import { render } from 'react-dom'
```

Comment:
Why are you importing ReactDOM AND render from react-dom? 
Assuming that you don't have a specific reason please remove the 
`import { render } from 'react-dom'` line as it is already covered by line 3